Makefile

```diff
@@ -5,12 +5,57 @@ Minversion := $(shell date)
 BUILD_NODE_PAR = -ldflags "-X github.com/ontio/ontology/common/config.Version=$(VERSION)" #-race
 BUILD_NODECTL_PAR = -ldflags "-X main.Version=$(VERSION)"
 
-all:
+ARCH=$(shell uname -m)
+DBUILD=docker build
+DRUN=docker run
+DOCKER_NS ?= ontio
+DOCKER_TAG=$(ARCH)-$(VERSION)
+ONT_CFG_IN_DOCKER=config-solo.json
+WALLET_FILE=wallet.dat
+
+all: ontology nodectl
+
+ontology:
 	$(GC)  $(BUILD_NODE_PAR) -o ontology main.go
+
+nodectl:
```

Comment:
nodectl has been removed in version v0.7.0

Reply:
I found 'make all' will still build nodectl.  
I'll update the pull-request when Makefile updated.